azure-samples/src/main/java/com/microsoft/azure/management/rediscache/samples/ManageRedisCache.java

```diff
@@ -0,0 +1,180 @@
+        final String redisCacheName2 = Utils.createRandomName("rc2");
+        final String redisCacheName3 = Utils.createRandomName("rc3");
+        final String rgName = Utils.createRandomName("rgRCMS");
+
+        try {
+
+            final File credFile = new File(System.getenv("AZURE_AUTH_LOCATION"));
+
+            Azure azure = Azure
+                    .configure()
+                    .withLogLevel(HttpLoggingInterceptor.Level.BASIC)
+                    .authenticate(credFile)
+                    .withDefaultSubscription();
+
+            // Print selected subscription
+            System.out.println("Selected subscription: " + azure.subscriptionId());
+
+            try {
+
+                // ============================================================
+                // Create a redis cache
+
+                System.out.println("Creating a Redis Cache");
+
+                RedisCache redisCache1 = azure.redisCaches().define(redisCacheName1)
+                        .withRegion(Region.US_CENTRAL)
+                        .withNewResourceGroup(rgName)
+                        .withBasicSku()
+                        .create();
+
+                System.out.println("Created a Redis Cache:");
+                Utils.print(redisCache1);
+
+
+                // ============================================================
+                // Get | regenerate Redis Cache access keys
+
+                System.out.println("Getting Redis Cache access keys");
+                RedisAccessKeys redisAccessKeys = redisCache1.keys();
+                Utils.print(redisAccessKeys);
+
+                System.out.println("Regenerating secondary Redis Cache access key");
+                redisAccessKeys = redisCache1.regenerateKey(RedisKeyType.SECONDARY);
+                Utils.print(redisAccessKeys);
+
+
+                // ============================================================
+                // Create another two Redis Caches
+
+                System.out.println("Creating two more Redis Caches with Premium Sku");
+
+                RedisCache redisCache2 = azure.redisCaches().define(redisCacheName2)
+                        .withRegion(Region.US_CENTRAL)
+                        .withNewResourceGroup(rgName)
+                        .withPremiumSku()
+                        .create();
+
+                System.out.println("Created a Redis Cache:");
+                Utils.print(redisCache2);
+
+                RedisCache redisCache3 = azure.redisCaches().define(redisCacheName3)
+                        .withRegion(Region.US_CENTRAL)
+                        .withNewResourceGroup(rgName)
+                        .withPremiumSku(2)
+                        .create();
+
+                System.out.println("Created a Redis Cache:");
+                Utils.print(redisCache3);
+                // ============================================================
+                // List Redis Caches
+
```

Comment:
Suggest that you update this comment with what is happening with Premium SKU inside the list walk